var/spack/repos/builtin/packages/netcdf/package.py

```diff
@@ -9,7 +9,7 @@ class Netcdf(Package):
     homepage = "http://www.unidata.ucar.edu/software/netcdf/"
     url      = "ftp://ftp.unidata.ucar.edu/pub/netcdf/netcdf-4.3.3.tar.gz"
 
-    version('4.4.0', 'f01cb26a0126dd9a6224e76472d25f6c')
+    version('4.4.0', 'cffda0cbd97fdb3a06e9274f7aef438e')
```

Comment:
Why did you need to change the md5sum of the package? As far as I can tell the old one is still correct.